api/src/services/organizations_v1/remove_user_from_organization.py

```diff
@@ -0,0 +1,126 @@
+import logging
+from uuid import UUID
+
+from sqlalchemy import select
+
+from src.adapters import db
+from src.api.route_utils import raise_flask_error
+from src.auth.endpoint_access_util import can_access
+from src.constants.lookup_constants import Privilege
+from src.db.models.entity_models import Organization
+from src.db.models.user_models import OrganizationUser, OrganizationUserRole, User
+from src.services.organizations_v1.get_organization import get_organization
+
+logger = logging.getLogger(__name__)
+
+
+def check_last_admin_protection(
+    db_session: db.Session, target_user_id: UUID, organization: Organization
+) -> None:
+    """Check if removing this user would leave the organization without any admins.
+
+    Args:
+        db_session: Database session
+        target_user_id: ID of user being removed
+        organization: Organization the user is being removed from
+
+    Raises:
+        FlaskError: 403 if this would remove the last admin
+    """
+    from src.db.models.user_models import LinkRolePrivilege, Role
+
+    # Get all user IDs in the organization who have.MANAGE_ORG_MEMBERS privilege
+    admin_user_ids_stmt = (
+        select(OrganizationUser.user_id)
+        .join(
+            OrganizationUserRole,
+            OrganizationUser.organization_user_id == OrganizationUserRole.organization_user_id,
+        )
+        .join(Role, OrganizationUserRole.role_id == Role.role_id)
+        .join(LinkRolePrivilege, Role.role_id == LinkRolePrivilege.role_id)
+        .where(OrganizationUser.organization_id == organization.organization_id)
+        .where(LinkRolePrivilege.privilege == Privilege.MANAGE_ORG_MEMBERS)
+    )
+
+    admin_user_ids = set(db_session.execute(admin_user_ids_stmt).scalars().all())
+
+    # Check if target user is an admin and if they're the only admin
+    if target_user_id in admin_user_ids and len(admin_user_ids) == 1:
+        raise_flask_error(403, message="Cannot remove the last administrator from organization")
+
+
+def validate_organization_user_exists(
```

Comment:
I think the update user service also has this function, could we move it to a central location and both of them use it?